.buildkite/scripts/steps/integration_tests.sh

```diff
@@ -19,7 +19,8 @@ fi
 
 # Run integration tests
 set +e
-AGENT_STACK_VERSION="${STACK_VERSION}" TEST_INTEG_CLEAN_ON_EXIT=true  STACK_PROVISIONER="$STACK_PROVISIONER" SNAPSHOT=true mage $MAGE_TARGET $MAGE_SUBTARGET
+# Remove AGENT_VERSION pinning as soon as the 9.0.0 is released
+AGENT_VERSION=9.0.0-SNAPSHOT AGENT_STACK_VERSION="${STACK_VERSION}" TEST_INTEG_CLEAN_ON_EXIT=true  STACK_PROVISIONER="$STACK_PROVISIONER" SNAPSHOT=true mage $MAGE_TARGET $MAGE_SUBTARGET
```

Comment:
here we are pinning AGENT_VERSION=9.0.0-SNAPSHOT but the AGENT_STACK_VERSION will be `9.1.0` due to the contents of .package-version, is this on purpose?